printer/printast_with_mappings.ml

```diff
@@ -151,6 +151,16 @@ let arg_label i ppf = function
 let typevars ppf vs =
   List.iter (fun x -> fprintf ppf " %a" Pprintast.tyvar x.txt) vs
 
+let modalities i ppf modes =
```

Comment:
This file is introduced by https://github.com/ocaml-flambda/flambda-backend/pull/155 but doesn't have tests, so we can't test our changes. However, it seems to be very similar to `printast.ml`. Therefore, could you simply try to unify this PR's changes to both files? (And unify my comments as well).